### PR TITLE
Add alerts

### PR DESCRIPTION
**Pull request by maya**
This adds alerts to the library :ballot_box_with_check: :x: :heavy_exclamation_mark: :information_source:

This addresses #240. 

This is what it looks like:

**Large screens**:
screen shot 2015-07-29 at 12 02 31 pm

**Small screens**:
screen shot 2015-07-29 at 12 15 05 pm

### CODE REVIEW

File: _components/alerts.md [hunk 1]
```diff
@@ -5,7 +5,72 @@ title: Alerts
 ---
 
 <div class="preview">
-  <!-- Add HTML markup for example here -->
+
+  <div class="usa-alert-success usa-sans">
+    <div class="usa-alert-icon">
+    <svg width="52" height="52" role="img" aria-label="success">
```
Comment (msecret) [inline _components/alerts.md]: I think this should be indendted another step

File: assets/_scss/components/_alerts.scss [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+.usa-alert-text {
+  margin: {
+    bottom: 0;
+    top: 0;
+  }
+}
+
+.usa-alert-success {
+  @extend .usa-alert;
```
Comment (msecret) [inline assets/_scss/components/_alerts.scss]: I actually think it's better to not do this and just include the `.usa-alert` class on each element. I remember a case where I tried to do exactly this for Peace corps, and I had to switch back for some reason (I can't remember the reason right now).

Comment (maya) [inline assets/_scss/components/_alerts.scss]: Ok seems like it could be safer. Updated here: 66b4a4b.

File: _components/alerts.md (hunk 1 above)
Comment (maya) [inline _components/alerts.md]: Fixed in 8b6035e.

File: _components/alerts.md [hunk 3]
```diff
@@ -5,8 +5,42 @@ title: Alerts
 ---
 
 <div class="preview">
-  <!-- Add HTML markup for example here -->
-  <img src="{{ site.baseurl }}/assets/img/static/Alerts_UI_v1.png">
+
+  <div class="usa-alert usa-alert-success usa-sans">
```
Comment (msecret) [inline _components/alerts.md]: Is there a reason why we can't put the usa-sans rule in usa-alert?

I feel like I see that class everywhere.

Comment (maya) [inline _components/alerts.md]: Sure, done in 6cb74bc.

**Pull request merged**